### PR TITLE
holaplex/holaplex/#347: high-res Twitter profile image and bug fix

### PR DESCRIPTION
**Pull request by austinmilt**
# Changes
- adds a high-resolution twitter pfp returned in the `profile` object for `wallet` requests (`profile.rs`)
- fixes query for `follow_wallets` to return wallets not already followed (and not self)

### CODE REVIEW

File: crates/graphql/src/schema/objects/profile.rs [hunk 1]
```diff
@@ -20,19 +21,17 @@ pub struct Profile {
 }
 
 impl From<TwitterUserProfileResponse> for TwitterProfile {
-    fn from(
-        TwitterUserProfileResponse {
-            screen_name,
-            description,
-            profile_image_url_https,
-            profile_banner_url,
-        }: TwitterUserProfileResponse,
```
Comment (kespinola) [inline crates/graphql/src/schema/objects/profile.rs:29]: We should keep the destructing of the struct.

**Review by kespinola — CHANGES_REQUESTED** [review]

File: crates/graphql/src/schema/objects/profile.rs (hunk 1 above)
Comment (austinmilt) [reply crates/graphql/src/schema/objects/profile.rs:29]: ah, yes, forgot that one. Thanks

**Review by ray-kast — APPROVED** [review]

**Comment by RlozanoR** [comment]
Let me get in with y'all

**Review by kespinola — APPROVED** [review]

**Comment by dandlezzz** [comment]
> Let me get in with y'all

send us a note, daniel@holaplex.com

**Pull request merged**